sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample10_SingleCategoryClassify.cs

```diff
@@ -19,7 +17,7 @@ public void SingleCategoryClassify()
             string endpoint = TestEnvironment.Endpoint;
             string apiKey = TestEnvironment.ApiKey;
 
-            var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            var client = CreateTextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
```

Comment:
I'm not a big fan of hiding the client creation in samples, to tell the truth. I wonder if this could be confusing to some users.

One thing I considered was creating a `TextAnalyticsClient` class inside the `Samples` namespace, and make it a child of the `Azure.AI.TextAnalytics.TextAnalyticsClient` class. We could keep the samples as they are, but we would be actually calling the constructor of our base client instead (and having it set the retry count for us).

I think this other approach is a bit sneaky and could be a source of bugs in the future, so I preferred to keep the clean one.

Reply:
Agreed.  I'm not a fan of this pattern.   Since all that we're doing in that method is specifying options, maybe we just generate the options there, rather than hiding the entire client creation?  

```csharp
#if SNIPPET
    var client = CreateTextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey),  CreateSampleOptions());
#else
    var client = CreateTextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
#endif
```

Even if we decide to keep `CreateTextAnalyticsClient` as-is, we should use the `SNIPPET` conditional to hide it from sample docs.

Reply:
Updated the samples with the suggested changes.